pkg/kn/commands/plugin/handler.go

```diff
@@ -93,6 +94,14 @@ func (h *DefaultPluginHandler) Lookup(name string) (string, bool) {
 
 // Execute implements PluginHandler
 func (h *DefaultPluginHandler) Execute(executablePath string, cmdArgs, environment []string) error {
+	if runtime.GOOS == "windows" {
+		cmd := exec.Command(executablePath, cmdArgs...)
+		cmd.Stdout = os.Stdout
```

Comment:
Could we do the same pattern for all runtime?

Reply:
I have read https://github.com/golang/go/issues/30662#issuecomment-470701281 and https://github.com/kubernetes/kubernetes/pull/76227/files#r284055366. 
The `exec` performs different  behaviors. On windows it fork a new process to run plugin . On Unix systems it starts a new executable in the same process.
Therefore I think it is hard to make the same pattern for all runtime. It may bring some unknown risks.

Reply:
Thanks. Still would love for someone to test this on Windows independently. Otherwise we’ll have to take your word for it. My guess is most everyone in WG are non-Windows users.

Reply:
Yes. We need test on different platforms. I have test it both on windows and linux.

Reply:
Nice thanks @MIBc will LGTM